lib/host/inject.js

```diff
@@ -1,11 +1,13 @@
 var injector = require('connect-injector');
 var debug = require('debug')('testee:html-injector');
+var path = require('path');
+var mime = require('mime-types');
 
 // Returns a middleware that injects the SocketIO JavaScript
 // and a link to the clients side adapters into any HTML file
 module.exports = function(configuration) {
   return injector(function(req, res) {
-    var header = res.getHeader('content-type');
+    var header = res.getHeader('content-type') || mime.contentType(path.extname(req.url));
```

Comment:
It has to use `path.extname`. It doesn't work without it and that's what is given in the [mime-types docs](https://www.npmjs.com/package/mime-types#mime-contenttype-type). I will need to handle URLs with query parameters; right now these are always returning false but the ```res.getHeader(...)``` call is working for these so it's ok. 

How ugly is this? ```mime.contentType(path.extname(req.url).split('?')[0]);```